esp/templates/program/modules/studentonsite/schedule.html

```diff
@@ -0,0 +1,80 @@
+<head>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link href="https://fonts.googleapis.com/css?family=Josefin+Sans" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="/media/styles/webapp.css" />
+</head>
+
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+<body>
+
+<div class="navbar">
+  <table>
+    <tr>
+      <td class="selected"><a class="no-dec" href="/learn/{{one}}/{{two}}/studentonsite/" style="display:block;"><i class="material-icons md-5">event_note</i>
+        <br>Schedule</td>
+      <td><a class="no-dec" href="/learn/{{one}}/{{two}}/onsitemap" style="display:block;"><i class="material-icons md-5">map</i>
+        <br>Map</a></td>
+      <td><a class="no-dec" href="/learn/{{one}}/{{two}}/onsitecatalog" style="display:block;"><i class="material-icons md-5">event</i>
+        <br>Catalog</td>
+      <td><a class="no-dec" href="/learn/{{one}}/{{two}}/onsitesurvey" style="display:block;"><i class="material-icons md-5">question_answer</i>
+        <br>Survey</td>
+    </tr>
+  </table>
+</div>
+
+<center>
+
+  <div class="main">
+    <table class="studentschedule">
+      <tbody>
```

Comment:
Maybe there should be a note here that if they are not yet checked in they cannot change around their courses? i.e.
{% if !checked_in %}
Note: You will not be able to change around your schedule till after your check-in has been processed by the Splash admin team
{% end if}
(or something less wordy?)

Reply:
Sounds pretty reasonable, I'll add this when I get a chance!